### PR TITLE
en: add doc for deploy across eks/gke

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added, or deleted? (Required)

add doc for deploy across eks/gke

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/1460
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by KanShiori — APPROVED** [review]

File: en/build-multi-gcp-gke.md [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+    gcloud compute networks subnets create ${subnet_3} \
+        --region=${region_3} \
+        --network=${network_name} \
+        --range=10.2.0.0/16 \
+        --secondary-range pods=10.12.0.0/16,services=10.102.0.0/16
+    ```
+
+    `${subnet_1}`, `${subnet_2}`, and `${subnet_3}` refer to the names of the three subnets.
+
+    `--range=10.0.0.0/16` specifies the CIDR block of the subnet in the cluster. The CIDR blocks of all cluster subnets **must not** overlap with each other.
```
Comment (qiancai) [inline en/build-multi-gcp-gke.md:74]:
```suggestion
    `--range=10.0.0.0/16` specifies the CIDR block of the `${subnet_1}` in the cluster. The CIDR blocks of all cluster subnets **must not** overlap with each other.
```

File: en/build-multi-gcp-gke.md [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+
+    `--secondary-range pods=10.11.0.0/16,services=10.101.0.0/16` specifies the CIRD block used by Kubernetes Pods and Services. This CIRD block will be used later.
+
+## Step 2. Start the Kubernetes cluster
+
+Create three GKE clusters, and each cluster uses the subnet created in Step 1.
```
Comment (qiancai) [inline en/build-multi-gcp-gke.md:80]:
```suggestion
Create three GKE clusters, and each cluster uses one of the subnets created in Step 1.
```

File: en/build-multi-gcp-gke.md [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+        ```
+
+        The expected output is as follows. The rule name is in the `NAME` column.
+
+        ```
+        NAME                           NETWORK     DIRECTION  PRIORITY  ALLOW                         DENY  DISABLED
+        gke-${cluster_1}-b8b48366-all  ${network}  INGRESS    1000      tcp,udp,icmp,esp,ah,sctp            False
+        ```
+
+    2. Update the source range of the firewall rule. Add the CIDR block of the Pod network of the other two clusters to the source range:
```
Comment (qiancai) [inline en/build-multi-gcp-gke.md:177]:
```suggestion
    2. Update the source range of the firewall rule. Add the CIDR blocks of the Pod network of the other two clusters to the source range:
```

File: en/build-multi-gcp-gke.md [hunk 4]
```diff
@@ -0,0 +1,291 @@
...
+
+    ```bash
+    kubectl --context ${context_1} -n default apply -f sample-nginx.yaml
+
+    kubectl --context ${context_2} -n default apply -f sample-nginx.yaml
+
+    kubectl --context ${context_3} -n default apply -f sample-nginx.yaml
+    ```
+
+3. Access the nginx services of other clusters to verify the network interconnectivity.
```
Comment (qiancai) [inline en/build-multi-gcp-gke.md:244]:
```suggestion
3. Access the nginx services of each cluster to verify the network interconnectivity.
```

File: en/build-multi-gcp-gke.md [hunk 5]
```diff
@@ -0,0 +1,291 @@
...
+    ```
+
+    If the output is the welcome page of nginx, the network is connected.
+
+4. After the verification, delete the nginx services:
+
+    {{< copyable "shell-regular" >}}
+
+    ```bash
+    kubectl --context ${context_2} -n default delete -f sample-nginx.yaml
```
Comment (qiancai) [inline en/build-multi-gcp-gke.md:261]:
```suggestion
    kubectl --context ${context_1} -n default delete -f sample-nginx.yaml
```

File: en/build-multi-aws-eks.md [hunk 6]
```diff
@@ -0,0 +1,426 @@
...
+
+- Grant AWS Access Key the [minimum permissions required for `eksctl`](https://eksctl.io/usage/minimum-iam-policies/) and the [permissions required for creating a Linux bastion](https://aws-quickstart.github.io/quickstart-linux-bastion/#_aws_account).
+
+To verify whether you have correctly configured the AWS CLI, run the `aws configure list` command. If the output shows the values of `access_key` and `secret_key`, you have successfully configured the AWS CLI. Otherwise, you need to reconfigure the AWS CLI.
+
+## Step 1. Start the Kubernetes cluster
+
+Define the configuration files of three EKS clusters as `cluster_1.yaml`, `cluster_2.yaml`, and `cluster_3.yaml`, and create three clusters using `eksctl`.
+
+1. Define the configuration file of Cluster 1, and create Cluster 1.
```
Comment (qiancai) [inline en/build-multi-aws-eks.md:34]:
```suggestion
1. Define the configuration file of cluster 1, and create cluster 1.
```

Comment (qiancai) [reply en/build-multi-aws-eks.md:34]: This is a global comment. It seems that we can use cluster 1, cluster 2, cluster 3 directly.

File: en/build-multi-aws-eks.md [hunk 7]
```diff
@@ -0,0 +1,426 @@
...
+        ```yaml
+        apiVersion: v1
+        kind: ConfigMap
+        # ...
+        data:
+          Corefile: |
+             .:53 {
+                # Do not modify the default configuration.
+             }
+             ${namspeace_2}.svc.cluster.local:53 {
```
[the remark this answers is not in the document]
Comment (qiancai) [reply en/build-multi-aws-eks.md:297]: Please make the same change in Chinese too.

File: en/deploy-tidb-cluster-across-multiple-kubernetes.md [hunk 8]
```diff
@@ -18,6 +18,8 @@ You need to configure the Kubernetes network and DNS so that the Kubernetes clus
 - The TiDB components on each Kubernetes cluster can access the Pod IP of all TiDB components in and between clusters.
 - The TiDB components on each Kubernetes cluster can look up the Pod FQDN of all TiDB components in and between clusters.
 
+To build multiple connected EKS or GKE clusters, refer to [Build Multiple Connected AWS EKS Clusters](build-multi-aws-eks.md) and [Build Multiple Connected GCP GKE Clusters](build-multi-gcp-gke.md).
```
Comment (qiancai) [inline en/deploy-tidb-cluster-across-multiple-kubernetes.md:21]:
```suggestion
To build multiple connected EKS or GKE clusters, refer to [Build Multiple Connected AWS EKS Clusters](build-multi-aws-eks.md) or [Build Multiple Connected GCP GKE Clusters](build-multi-gcp-gke.md).
```

**Review by qiancai — COMMENTED** [review]
@ran-huang I've completed the review and left some comments. PTAL

File: en/build-multi-aws-eks.md (hunk 6 above)
Comment (ran-huang) [reply en/build-multi-aws-eks.md]: Thanks. Done!

File: en/build-multi-aws-eks.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (ran-huang) [reply en/build-multi-aws-eks.md]: Fixed.

File: en/build-multi-aws-eks.md [hunk 9]
```diff
@@ -0,0 +1,426 @@
...
+    | `${cidr_block_2}` | `${vpc_peering_id_12}` | Active | No         |
+    | `${cidr_block_3}` | `${vpc_peering_id_13}` | Active | No         |
+
+    The **Destination** of each route is the CIDR block of another cluster. The **Target** is the VPC peering ID of the two clusters.
+
+### Update the security groups for the instances
+
+1. Update the security group for Cluster 1.
+
+    1. Enter the AWS Security Groups Console and select the security group of Cluster 1. The name of the security group is similar to `eksctl-${cluster_1}-cluster/ClusterSharedNodeSecurityGroup`.
```
[the remark this answers is not in the document]
Comment (ran-huang) [reply en/build-multi-aws-eks.md:151]: Added. Thanks!

**Comment by ran-huang** [comment]
@qiancai All comments addressed. PTAL again.

File: en/build-multi-aws-eks.md [hunk 10]
```diff
@@ -0,0 +1,426 @@
...
+
+    ```bash
+    lb_name_1=$(kubectl --context ${context_1} -n kube-system get svc across-cluster-dns-tcp -o jsonpath="{.status.loadBalancer. ingress[0].hostname}" | cut -d - -f 1)
+
+    lb_name_2=$(kubectl --context ${context_2} -n kube-system get svc across-cluster-dns-tcp -o jsonpath="{.status.loadBalancer. ingress[0].hostname}" | cut -d - -f 1)
+
+    lb_name_3=$(kubectl --context ${context_3} -n kube-system get svc across-cluster-dns-tcp -o jsonpath="{.status.loadBalancer. ingress[0].hostname}" | cut -d - -f 1)
+    ```
+
+    Check the load balancer status of each cluster by running the following commands. If the output of all commands is "active", all load balancers are in the `Active` state.
```
Comment (qiancai) [inline en/build-multi-aws-eks.md:217]:
```suggestion
    Check the load balancer status of each cluster by running the following commands. If the output of all commands is `active`, all load balancers are in the `Active` state.
```

**Review by qiancai — APPROVED** [review]

**Comment by qiancai** [comment]
LGTM

**Pull request merged**